Modules/IO/GDCM/src/itkGDCMImageIO.cxx

```diff
@@ -331,6 +333,33 @@ GDCMImageIO::Read(void * pointer)
     // WARNING: sizeof(Real World Value) != sizeof(Stored Pixel)
     len = len * outputpt.GetPixelSize() / pixeltype.GetPixelSize();
   }
+  /*
+   * YBR to RGB
+   * Currently (GDCM 3.0.2) processing for lossy Jpeg YBR_FULL_422
+   * is not fully clear. May be later the files will be converted
+   * internally in GDCM, in that case remove YBR_FULL_422 below.
+   *
+   * TODO: UINT16
+   *
+   */
+  if (m_NumberOfComponents == 3 &&
+      (pi == gdcm::PhotometricInterpretation::YBR_FULL || pi == gdcm::PhotometricInterpretation::YBR_FULL_422) &&
```

Comment:
You're doing twice YBR_FULL_422 -> RGB. I suspect this is wrong, please test with JPEG Lossy 8 bits this chunk.

Reply:
Could you add a comment here, explaining how JPEG Lossy/YBR_FULL_422 is not re-entering this loop ? Thanks much

Reply:
@malaterre  there is no loop, please kindly test

Reply:
@malaterre  If you  [comment two lines here in GDCM,](https://github.com/InsightSoftwareConsortium/ITK/blob/3526b67168ded206a8b9a3e7100fb1849c476d8e/Modules/ThirdParty/GDCM/src/gdcm/Source/MediaStorageAndFileFormat/gdcmJPEGBITSCodec.hxx#L861) convert wiil be not required for YBR_FULL_422, i have tried PR in GDCM and closed, please see yourself, may be it will have impact on some GDCM tools or whatever

```
      if( GetPhotometricInterpretation() == PhotometricInterpretation::YBR_FULL
      || GetPhotometricInterpretation() == PhotometricInterpretation::YBR_FULL_422 )
        {
// <------  cinfo.jpeg_color_space = JCS_UNKNOWN;
// <------  cinfo.out_color_space = JCS_UNKNOWN;
        //this->PlanarConfiguration = 1;
        }
```